packages/graphql/src/translate/authorization/utils/filter-by-values.ts

```diff
@@ -0,0 +1,132 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { whereRegEx } from "../../where/utils";
+import type { WhereRegexGroups } from "../../where/utils";
+
+type ComparatorFn<T> = (received: T, filtered: T) => boolean;
+
+const operatorCheckMap = {
+    NOT: (received: string, filtered: string) => received !== filtered,
+    LT: (received: number | string, filtered: number | string) => {
+        return received < filtered;
+    },
+    LTE: (received: number, filtered: number) => {
+        return received <= filtered;
+    },
+    GT: (received: number, filtered: number) => {
+        return received > filtered;
+    },
+    GTE: (received: number | string, filtered: number | string) => {
+        return received >= filtered;
+    },
+    STARTS_WITH: (received: string, filtered: string) => received.startsWith(filtered),
+    NOT_STARTS_WITH: (received: string, filtered: string) => !received.startsWith(filtered),
+    ENDS_WITH: (received: string, filtered: string) => received.endsWith(filtered),
+    NOT_ENDS_WITH: (received: string, filtered: string) => !received.endsWith(filtered),
+    CONTAINS: (received: string, filtered: string) => received.includes(filtered),
+    NOT_CONTAINS: (received: string, filtered: string) => !received.includes(filtered),
+    INCLUDES: (received: [string | number], filtered: string | number) => {
+        return received.findIndex((v) => v === filtered) !== -1;
+    },
+    NOT_INCLUDES: (received: [string | number], filtered: string | number) => {
+        return received.findIndex((v) => v === filtered) === -1;
+    },
+    IN: (received: string | number, filtered: [string | number]) => {
+        return filtered.findIndex((v) => v === received) !== -1;
+    },
+    NOT_IN: (received: string | number, filtered: [string | number]) => {
+        return filtered.findIndex((v) => v === received) === -1;
+    },
+};
+function getFilteringFn<T>(operator: string | undefined): ComparatorFn<T> {
+    if (!operator) {
+        return (received: T, filtered: T) => received === filtered;
+    }
+    return operatorCheckMap[operator];
+}
+
+function parseFilterProperty(key: string): { fieldName: string; operator: string | undefined } {
+    const match = whereRegEx.exec(key);
+    if (!match) {
+        throw new Error(`Failed to match key in filter: ${key}`);
+    }
+    const { fieldName, operator } = match.groups as WhereRegexGroups;
+    if (!fieldName) {
+        throw new Error(`Failed to find field name in filter: ${key}`);
+    }
+    return { fieldName, operator };
+}
```

Comment:
Unless I've misread, this seems to be duplicated code from packages/graphql/src/schema/resolvers/subscriptions/where/utils/parse-filter-property.ts

Can we deduplicate?